src/components/Header/ListenerWidth.jsx

```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from 'react'
+
+function getWindowDimensions() {
+    const width = window.innerWidth
+    return width
+}
+export default function ListenerWidth() {
```

Comment:
you have created your own hook! Cool! But general note, there is a convention to name hooks with use prefix. Like useScreenWidth

Reply:
A very very good note. A very good lesson for me. Thanks, I'll fix it )))